lib/migration-stream.js

```diff
@@ -29,32 +32,70 @@ function MigrationStream() {
     });
 
     if (v2 === true) {
-      // Check for errors on the v2 tile
-      var err = checkForErrors(info);
-      if (err) return callback(err);
-
-      migrationStream.push(tile);
-      return callback();
-    } else {
+      decompressBuffer(tile.buffer, function(err, decompressedBuffer){
+          if (err) {
+            err.code = 'EINVALID';
+            return callback(err);
+          } 
+
+          vtvalidate.isValid(decompressedBuffer, function(err, result){
```

Comment:
Flagging youll still need to handle `err` here

Reply:
fixed!